auth0/src/test/java/com/auth0/android/management/UsersAPIClientTest.java

```diff
@@ -128,14 +129,16 @@ public void tearDown() throws Exception {
     @Test
     public void shouldUseCustomNetworkingClient() throws IOException {
         Auth0 account = new Auth0("client-id", "https://tenant.auth0.com/");
-        InputStream inputStream = new StringInputStream("{\"id\":\"undercover\"}");
+        String jsonResponse = "{\"id\":\"undercover\"}";
+        InputStream inputStream = new ByteArrayInputStream(jsonResponse.getBytes());
         ServerResponse response = new ServerResponse(200, inputStream, Collections.emptyMap());
         NetworkingClient networkingClient = mock(NetworkingClient.class);
         when(networkingClient.load(anyString(), any(RequestOptions.class))).thenReturn(response);
         UsersAPIClient client = new UsersAPIClient(account, "token.token", networkingClient);
 
         Request<UserProfile, ManagementException> request = client.getProfile("undercover");
         request.execute();
+        ShadowLooper.idleMainLooper();
```

Comment:
This line is required to be run after every request is executed or scheduled to be run. Because Robolectric blocks the thread until we say "idle", when it will release the lock and continue the execution.